internal/appfunction/context_test.go

```diff
@@ -442,6 +448,199 @@ func TestContext_GetDeviceResource_Error(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestContext_Publish(t *testing.T) {
+	tests := []struct {
+		name            string
+		message         string
+		publishErr      error
+		commonConfig    *appCommon.ConfigurationStruct
```

Comment:
Common config has a specific meaning now ;-)
```suggestion
		triggerConfig    *appCommon.ConfigurationStruct
```